### ISSUE TITLE
anonymously uploaded data never shows up in map

### ISSUE THREAD

**Issue by TjrGithub** [openbmap/radiocells-scanner-android#187]
What steps will reproduce the problem?
Install radiobeacon from fdroid on a slimrom6 device, don't configure a username/password but instead activate anonymous upload, go mapping, upload data.

What is the expected output? What do you see instead?
I expect the mapped cell towers and wlans to appear on the map on the openbmap website the next day. Instead, there is absolutely nothing in my target area, several weeks later. The Radiobeacon app did go through all the usual uploading data spiel, and it did find several hundred wifis etc.

What version are you using? On what operating system?
Version 0.8.18(ef93348-dirty) from fdroid. Android 6.0.1, Firmware Slim-6.0.1 kenzo

Please provide any additional information below.
Uploading by itself works: in an earlier install of the same setup, I successfully uploaded under the username tjb and found myself on the monthly highscore list. On the affected mapping tour, the localwifinlpbackend found a few thousand new wifis, and the osmand gpx track file also looks good.

**Comment by wish7code** [comment]
Could you share the approximate area for debugging purposes?

**Comment by gdt** [comment]
Also, there's "never" vs "until someone(tm) gets around to a manual database processing run".  So it would be really nice if the website showed "data last updated as of [timestamp],  most recent data point [timestamp]".

**Comment by TjrGithub** [comment]
I mapped these two neighborhoods: 
http://www.openstreetmap.org/#map=17/50.89217/11.60775
http://www.openstreetmap.org/#map=17/50.88236/11.60776

Today, my tracks finally showed up on the map on https://radiocells.org/stats/countries. Yesterday night, they were neither on that map nor in Radiobeacon, even though I downloaded the de.sqlite purportedly generated on 2017-02-22.

**Comment by wish7code** [comment]
Glad to hear! Guess someone (tm) has pushed the button as Greg suspected ;-)

Seriously, automatic wifi import had a hickup, cell import usually takes place only once a week.
Please reopen issue, if you're still missing data!

![image](https://cloud.githubusercontent.com/assets/8270020/23321910/d39b3cd0-fae1-11e6-9184-f01ff13fb108.png)

![image](https://cloud.githubusercontent.com/assets/8270020/23321998/3a23fbcc-fae2-11e6-8ddf-a8a2275fdb9f.png)

Greetings to Jena!

**Comment by wish7code** [comment]
@gdt Makes perfect sense, created a separate issue #189 regarding the import date

**Comment by TjrGithub** [comment]
I've removed identifying information from my comments for legal reasons, and I'd appreciate if you removed the map, too.

**Comment by wish7code** [comment]
Done..

Feel free to use wish7code@gmail or team@radiocells for private communication!
